apps/articles/test/client/article.coffee

```diff
@@ -91,10 +102,36 @@ describe 'ArticleIndexView', ->
       @view.render(@view.collection, results: articles )
       $('.article-container').length.should.equal 2
 
-  describe '#initFixedShare', ->
+  describe 'promoted content gallery', ->
 
-    it 'renders the share icons', ->
-      @view = new @ArticleIndexView el: $('body')
-      $('.article-fixed-share').html().should.containEql 'https://twitter.com/intent/tweet?original_referer=undefined&amp;text=Top Ten Booths at miart 2014 @artsy&amp;url=undefined/article/undefined'
-      $('.article-fixed-share').html().should.containEql 'https://www.facebook.com/sharer/sharer.php?u=undefined/article/undefined'
-      $('.article-fixed-share').html().should.containEql 'mailto:?subject=Top Ten Booths at miart 2014 @artsy&amp;body=Check out Top Ten Booths at miart 2014 @artsy on Artsy: undefined/article/undefined'
+    before (done) ->
+      @options.sd.PC_ARTSY_CHANNEL = '5086df098523e60002000011'
+      @ArticleIndexView.__set__ 'sd', { PC_ARTSY_CHANNEL: '5086df098523e60002000011', ARTICLE: @model.attributes }
+      benv.render resolve(__dirname, '../../templates/article.jade'), @options, =>
+        @view = new @ArticleIndexView
+          el: $('body')
+        done()
+
+    it 'displays promoted content banner for partner', ->
+
+      Backbone.sync.args[0][2].success fabricate 'partner'
+      Backbone.sync.args[4][2].success fabricate 'partner_profile'
+      $('.articles-promoted').attr('href').should.equal '/getty'
+      $('.articles-promoted__name').text().should.equal 'Gagosian Gallery'
+
+  describe 'promoted content auction', ->
+
+    before (done) ->
+      @options.sd.PC_AUCTION_CHANNEL = '5086df098523e60002000011'
+      @options.sd.PC_ARTSY_CHANNEL = '5086df098523e60002000012'
+      @ArticleIndexView.__set__ 'sd', { PC_AUCTION_CHANNEL: '5086df098523e60002000011', PC_ARTSY_CHANNEL: '5086df098523e60002000012', ARTICLE: @model.attributes }
+      benv.render resolve(__dirname, '../../templates/article.jade'), @options, =>
+        @view = new @ArticleIndexView
+          el: $('body')
+        done()
+
+    it 'displays promoted content banner for auction', ->
+
+      Backbone.sync.args[5][2].success fabricate 'sale'
```

Comment:
In refactoring realized the error case was in mind of force, not relevant in microgravity -- ended up removing the error callbacks.